braph2genesis/measures/_OutGlobalEfficiency.gen.m

```diff
@@ -0,0 +1,286 @@
+   [BRAPH2.STR ':OutGlobalEfficiency:' BRAPH2.FAIL_TEST], ...
+    [class(m_outside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+m_inside_g = g.get('MEASURE', 'OutGlobalEfficiency');
+assert(isequal(m_inside_g.get('M'), known_out_global_efficiency), ...
+    [BRAPH2.STR ':OutGlobalEfficiency:' BRAPH2.FAIL_TEST], ...
+    [class(m_inside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+%%% ¡test!
+%%%% ¡name!
+MultiplexBD
+%%%% ¡code!
+B11 = [
+      0   .1  0   0   0
+      .2  0   0   0   0
+      0   0   0   .2  0
+      0   0   .1  0   0
+      0   0   0   0   0
+      ];
+
+B22 = [
+      0   .1  0   0   0
+      .2  0   0   0   0
+      0   0   0   .2  0
+      0   0   .1  0   0
+      0   0   0   0   0
+      ];
+B = {B11  B22};
+
+known_out_global_efficiency = {
+                        [1/4 1/4 1/4 1/4 0]'
+                        [1/4 1/4 1/4 1/4 0]'
+                        };
+
+g = MultiplexBD('B', B);
+
+m_outside_g = OutGlobalEfficiency('G', g);
+assert(isequal(m_outside_g.get('M'), known_out_global_efficiency), ...
+   [BRAPH2.STR ':OutGlobalEfficiency:' BRAPH2.FAIL_TEST], ...
+    [class(m_outside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+m_inside_g = g.get('MEASURE', 'OutGlobalEfficiency');
+assert(isequal(m_inside_g.get('M'), known_out_global_efficiency), ...
+    [BRAPH2.STR ':OutGlobalEfficiency:' BRAPH2.FAIL_TEST], ...
+    [class(m_inside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+%%% ¡test!
+%%%% ¡name!
+OrdMxWD
+%%%% ¡code!
+B11 = [
+      0   .1  0   0   0
+      .2  0   0   0   0
+      0   0   0   .2  0
+      0   0   .1  0   0
+      0   0   0   0   0
+      ];
+
+B22 = [
+      0   .1  0   0   0
+      .2  0   0   0   0
+      0   0   0   .2  0
+      0   0   .1  0   0
+      0   0   0   0   0
+      ];
+B = {B11  B22};
+
+known_out_global_efficiency = {
+                        [.1/4 .2/4 .2/4 .1/4 0]'
+                        [.1/4 .2/4 .2/4 .1/4 0]'
+                        };
+
+g = OrdMxWD('B', B);
+
+m_outside_g = OutGlobalEfficiency('G', g);
+assert(isequal(m_outside_g.get('M'), known_out_global_efficiency), ...
+   [BRAPH2.STR ':OutGlobalEfficiency:' BRAPH2.FAIL_TEST], ...
+    [class(m_outside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+m_inside_g = g.get('MEASURE', 'OutGlobalEfficiency');
+assert(isequal(m_inside_g.get('M'), known_out_global_efficiency), ...
+    [BRAPH2.STR ':OutGlobalEfficiency:' BRAPH2.FAIL_TEST], ...
+    [class(m_inside_g) ' is not being calculated correctly for ' class(g) '.'])
+
+%%% ¡test!
+%%%% ¡name!
+OrdMxBD
+%%%% ¡code!
+B11 = [
+      0   .1  0   0   0
+      .2  0   0   0   0
+      0   0   0   .2  0
+      0   0   .1  0   0
+      0   0   0   0   0
+      ];
+
+B22 = [
+      0   .1  0   0   0
+      .2  0   0   0   0
+      0   0   0   .2  0
+      0   0   .1  0   0
+      0   0   0   0   0
+      ];
+B = {B11  B22};
+
+known_out_global_efficiency = {
```

Comment:
Same, here I think it need to be the transpose (so we have to add ' to the end. Like this: known_out_global_efficiency = {[1/4 1/4 1/4 1/4 0]'
[1/4 1/4 1/4 1/4 0]'};